packages/cli-core/README.md

```diff
@@ -0,0 +1,3 @@
+# @loopback/cli-core
```

Comment:
> We should allow different CLI commands to be packaged in their own modules and contribute to the core as a component.

We already have 26 entries under `/packages` directory, I feel that's already too much.

`@loopback/cli` has 18 commands (generators) so far. If we put all per-command packages to `packages/`, we will end up with 44+ entries 😱

Let's create a dedicated top-level directory for all new CLI packages please. I am proposing to use the directory name `cli`, e.g. `/cli/cli-core`, `/cli/model`, `/cli/repository`, etc.